### ISSUE TITLE
I understand this launcher is for Russian speaking but can you add English translations?

### ISSUE THREAD

**Issue by RDKRACZ** [GravitLauncher/Launcher#516]

**Comment by Limon4ik66607** [comment]
If you are talking about the launcher itself, then in the settings you can switch the language.

**Comment by RDKRACZ** [comment]
I meant the website and future commits if possible (not all of them are in English currently)

**Comment by gravit0** [comment]
Translating documentation is not easy. Google Translate to translate the documentation is very bad?

**Comment by RDKRACZ** [comment]
It's better than nothing and most parts are understandable but an official translation would be preferred. I understand if you can't do it yet.
Also, can you add some screenshots to the wiki and readme? It also helps and I feel like projects look more professional then.

**Comment by MaximusMcfly** [comment]
@gravit0 Если тебе самому лень переводить ты можешь открыть перевод на https://crowdin.com/  и люди переведут его для тебя
-----ENGLISH VERSION-----
@gravit0 If you're too lazy to translate it by yourself then you can open translation on https://crowdin.com/ and pepole will translate it for you

**Comment by daniorio** [comment]
You can use https://www.deepl.com for correct translate. It is the best translater.

**Comment by MaximusMcfly** [comment]
Yes, but even the best translate software/website has its own flaws.
